.gitignore

```diff
@@ -24,5 +24,7 @@ _testmain.go
 *.prof
 .env
 
-coverage.out
+*.swp
+coverage*.out
+coverage.xml
```

Comment:
Yeah, I can remove this. Cobertura uses xml files so I added a few on my local environment for testing, but it doesn't need to go on .gitignore, my bad.